common/src/test/java/org/jboss/pnc/common/json/moduleconfig/JenkinsBuildDriverModuleConfigTest.java

```diff
@@ -10,12 +10,11 @@
     @Test
     public void test() {
```

Comment:
Ok, since we got the survey results, please refactor this test also:
* Add some meaningful name
* Divide into 3 blocks //given //when //then
* Get rid of this nasty catch block